openmc/deplete/results.py

```diff
@@ -460,6 +463,25 @@ def get_times(self, time_units: str = "d") -> np.ndarray:
 
         return _get_time_as(times, time_units)
 
+    def get_source_rates(self) -> np.ndarray:
+        """
+        .. versionadded:: 0.15.1
+
+        Returns
+        -------
+        numpy.ndarray
+            1-D vector of source rates at each point in the depletion simulation
+            with the units originally defined by the user.
+
+        """
+        source_rates = np.fromiter(
+            (r.source_rate for r in self),
+            dtype=self[0].source_rate.dtype,
+            count=len(self)-1,
+        ) # Results duplicates the final source rate at the final simulation time
```

Comment:
Let's move this comment above the code.

```suggestion
        ) # Results duplicate the final source rate at the final simulation time
```